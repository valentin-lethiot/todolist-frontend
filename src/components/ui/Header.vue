<script setup lang="ts"></script>

<template>
  <header class="flex flex-col">
    <h1 class="text-3xl font-bold text-center">Qu'allons-nous faire aujourd'hui ?</h1>
    <hr class="my-6 h-0.5 w-40 w-full bg-white rouded-lg" />
  </header>
</template>
