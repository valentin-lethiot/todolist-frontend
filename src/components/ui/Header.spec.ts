import { describe, beforeEach, expect, it } from 'vitest'
import Header from './Header.vue'
import { mount, VueWrapper } from '@vue/test-utils'

describe('Header', () => {
  let wrapper: VueWrapper

  beforeEach(() => {
    wrapper = mount(Header)
  })

  describe('render', () => {
    it('should render the header', () => {
      expect(wrapper.html()).toMatchSnapshot()
    })
  })
})
