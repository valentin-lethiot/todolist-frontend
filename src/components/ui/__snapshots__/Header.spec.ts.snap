// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Header > render > should render the header 1`] = `
"<header class="flex flex-col">
  <h1 class="text-xl font-bold text-center md:text-3xl">Qu'allons-nous faire aujourd'hui ?</h1>
  <hr class="my-2 md:my-6 h-0.5 w-40 w-full bg-white rounded-lg">
</header>"
`;
